keystore/Cargo.toml

```diff
@@ -57,10 +57,9 @@ git = "https://github.com/wireapp/proteus"
 branch = "2.x"
 
 [target.'cfg(not(target_family = "wasm"))'.dependencies]
-async-fs = "2.0"
-blocking = "1.2"
-# Fixes the dynamic linking issue on Android with the atomic lib
-openssl-sys = "=0.9.92"
+async-fs = "2.1"
+blocking = "1.6"
+openssl-sys = "0.9"
```

Comment:
Is there a reason why we specify `openssl-sys` here explicitly? I don't think we are using it directly, but via one of our other dependencies so it  would be best to let those dependencies pick the version for us (and hopefully make dep resolver's work easier).